composer.json

```diff
@@ -3,7 +3,7 @@
     "description": "Better Reflection - an improved code reflection API",
     "license": "MIT",
     "require": {
-        "php": "^7.0|^7.1",
+        "php": "^7.0",
```

Comment:
Hrmm, I know it's not technically needed, but I think it helps; especially when `php` itself isn't actually semver. In fact, perhaps better to be `7.0.*|7.1.*` heh?